stubs/Pillow/PIL/ImageMorph.pyi

```diff
@@ -1,24 +1,26 @@
-from typing import Any
+from _typeshed import SupportsRead
 
-LUT_SIZE: Any
-ROTATION_MATRIX: Any
-MIRROR_MATRIX: Any
+from .Image import Image
+
+LUT_SIZE: int
+ROTATION_MATRIX: list[int]
+MIRROR_MATRIX: list[int]
 
 class LutBuilder:
-    patterns: Any
-    lut: Any
-    def __init__(self, patterns: Any | None = ..., op_name: Any | None = ...) -> None: ...
-    def add_patterns(self, patterns) -> None: ...
+    patterns: list[str]
+    lut: bytearray
+    def __init__(self, patterns: list[str] | None = ..., op_name: str | None = ...) -> None: ...
+    def add_patterns(self, patterns: list[str]) -> None: ...
     def build_default_lut(self) -> None: ...
-    def get_lut(self): ...
-    def build_lut(self): ...
+    def get_lut(self) -> bytearray: ...
+    def build_lut(self) -> bytearray: ...
 
 class MorphOp:
-    lut: Any
-    def __init__(self, lut: Any | None = ..., op_name: Any | None = ..., patterns: Any | None = ...) -> None: ...
-    def apply(self, image): ...
-    def match(self, image): ...
-    def get_on_pixels(self, image): ...
-    def load_lut(self, filename) -> None: ...
-    def save_lut(self, filename) -> None: ...
-    def set_lut(self, lut) -> None: ...
+    lut: bytearray
+    def __init__(self, lut: bytearray | None = ..., op_name: str | None = ..., patterns: list[str] | None = ...) -> None: ...
+    def apply(self, image: Image) -> tuple[int, Image]: ...
+    def match(self, image: Image) -> list[tuple[int, int]]: ...
+    def get_on_pixels(self, image: Image) -> list[tuple[int, int]]: ...
+    def load_lut(self, filename: str | bytes | SupportsRead[bytes]) -> None: ...
+    def save_lut(self, filename: str | bytes | SupportsRead[bytes]) -> None: ...
```

Comment:
```suggestion
    def load_lut(self, filename: StrOrBytesPath) -> None: ...
    def save_lut(self, filename: StrOrBytesPath) -> None: ...
```
These two both just do `open()`.

Reply:
Isn't what I wrote what `read` accepts for the first argument? E.g. https://github.com/python/typeshed/blob/d3ba8a38587ceb8c54dfa9fed32c1028aa80595e/stubs/Pillow/PIL/ImageFont.pyi#L16 or https://github.com/python/typeshed/blob/d3ba8a38587ceb8c54dfa9fed32c1028aa80595e/stubs/Pillow/PIL/ImageFont.pyi#L23 (and other places in typeshed with this list of things).

Reply:
These functions just call builtin `open`, not `Image.read`.

Reply:
Sure; I was just thinking of this:

https://github.com/python/typeshed/blob/389b92cb04573072dfcb5c072ec1c59b218e664e/stdlib/builtins.pyi#L1149

Which would imply that I should have written `StrOrBytesPath | int`.

Reply:
(My original copy/paste was wrong, for some reason.)

Reply:
`int` would also work I guess, though opening an fd is a pretty rare operation.

In any case, `StrOrBytesPath` is better than the current union involving `SupportsRead`.

Reply:
Yes, agreed; not sure what I was thinking there.